### PR TITLE
Jm call eui assets

### PR DESCRIPTION
**Pull request by kjaymiller**
Includes:
   * readme update from @brittanyjoiner15
   * header and single object template

### CODE REVIEW

**Comment by brittanyjoiner15** [comment]
@kjaymiller did you commit the `@elastic/eui` pieces to the `yarn.lock` or `package.json` ? I'm not seeing them in there, so when i run yarn, it doesn't install them and then throws and error about needing those dependencies.

**Comment by brittanyjoiner15** [comment]
@kjaymiller just pushed some changes - i think if we just update `package.json` with those lines, that should work! once added those, then ran `yarn install`, i was able to successfully run this!

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,9838 @@
...
+    postcss-value-parser "^4.2.0"
+
+"@elastic/datemath@^5.0.3":
+  version "5.0.3"
+  resolved "https://registry.yarnpkg.com/@elastic/datemath/-/datemath-5.0.3.tgz#7baccdab672b9a3ecb7fe8387580670936b58573"
+  integrity sha512-8Hbr1Uyjm5OcYBfEB60K7sCP6U3IXuWDaLaQmYv3UxgI4jqBWbakoemwWvsqPVUvnwEjuX6z7ghPZbefs8xiaA==
+  dependencies:
+    tslib "^1.9.3"
+
+"@elastic/eui@^48.1.1":
```
Comment (kjaymiller) [inline yarn.lock]: Looks like it's in here. Only about 1100 lines in 😉

**Comment by kjaymiller** [comment]
I have a further update with some static implementation of the `Terry Stops` page with some examples for cards. I was running into issues with Elastic-Charts not rendering (I think we have to do something with typescript. Or maybe we can ask someone about it.

I think pushing this will get you up to about where I got over the weekend but I couldn't figure anything for the charts and definitely think we may need to look at how to link to data structures a little easier than manual content.

**Pull request merged**